kas-project.yml

```diff
@@ -8,10 +8,10 @@ repos:
     meta-hulks:
     meta-nao:
         url: "https://github.com/HULKs/meta-nao.git"
-        refspec: 68ebdbf38569d60792337fea2af932f389e7497e
+        refspec: 5650dafedddb4c67f1ef7cd5aa7a059da228040d
     poky:
         patches:
-            0001-Rust-1.70:
```

Comment:
Do we want to merge an outdated Rust version? Current version is 1.71... :wink:

Reply:
As discussed, let's merge this for now and upgrade to Rust ~~1.71~~ 1.72 afterwards.